### PR TITLE
Done

### PR DESCRIPTION
**Pull request by talmasica**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -134,6 +134,13 @@ a:visited {
   height: 400px;
 }
 
+#tal-masica-cover {
+  background-image: url(http://i.imgur.com/wkRJr7k.jpg);
```
Comment (bhollan) [inline css/styles.css]: If you look at the other examples, you'll see `""`s around all the image URLs.  Yours should conform to the norm.

File: css/styles.css [hunk 2]
```diff
@@ -134,6 +134,13 @@ a:visited {
...
+}
+#tal-masica-card {
+  background-image: url(http://i.imgur.com/Nizjqhg.jpg);
```
Comment (bhollan) [inline css/styles.css]: Same here.

**Comment by bhollan** [comment]
Hi @talmasica !

It looks like you've forgotten to change `index.html` and `css/styles.css`.  Take a look at the lessons and see if you can get those changes committed and push.

If you need any help, feel free to Ask A Question about this lesson!

CC: @aturkewi

**Comment by talmasica** [comment]
think i did it. thanks

**Comment by bhollan** [comment]
If you click on where it says "[Files changed: 3](https://github.com/learn-co-students/learn-deploy-on-day-one-v-000/pull/660/files)" above, you'll see exactly what I see when I review your changes.  If you have made your changes locally, I won't be able to see them here until they are `commit`ted and `push`ed.

Feel free to Ask A Question if you need help!

**Comment by bhollan** [comment]
Hi @talmasica !

I still don't see the changes on github.  That means the changes haven't been saved, committed, or pushed, or some combination of the three.

As always, feel free to Ask A Question about this lab if you need help figuring it out!

CC: @aturkewi

**Comment by bhollan** [comment]
Hi @talmasica !

I'm going to close this PR for now.  We can re-open it any time you like.  Just let us know whenever you're ready to implement the necessary changes.

If you need any help, as always, please feel free to Ask A Question!

CC: @aturkewi

**Pull request closed without merging**